src/test/java/com/commercetools/sync/inventories/InventorySyncMockUtils.java

```diff
@@ -84,12 +84,12 @@ public static InventoryEntry getMockInventoryEntry(final String sku,
      * @return mock instance of {@link InventoryService}
      */
     static InventoryService getMockInventoryService(final Set<InventoryEntry> inventoryEntries,
-                                                    final InventoryEntry createdInventoryEntry,
+                                                    final Optional<InventoryEntry> createdInventoryEntry,
```

Comment:
Why did you make this change? Why not just passing null instead?

Reply:
c4af294d